command/rootchain/initcontracts/init_contracts.go

```diff
@@ -0,0 +1,321 @@
+package initcontracts
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"math/big"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/umbracle/ethgo"
+	"github.com/umbracle/ethgo/abi"
+
+	"github.com/0xPolygon/polygon-edge/command"
+	"github.com/0xPolygon/polygon-edge/command/genesis"
+	"github.com/0xPolygon/polygon-edge/command/rootchain/helper"
+	"github.com/0xPolygon/polygon-edge/contracts"
+	"github.com/0xPolygon/polygon-edge/helper/hex"
+	"github.com/0xPolygon/polygon-edge/types"
+)
+
+var (
+	params initContractsParams
+
+	initCheckpointManager, _ = abi.NewMethod("function initialize(" +
```

Comment:
Is there an error check?
If yes - could you move initialisation to init function and add error checking?

Reply:
This can throw error only if we don't specify method signature correctly (so it can't in our case). Maybe we should add some linter flag or something?

Reply:
No need for adding linter ignore, because linter hasn't reported a warning. We deliberately ignored error handling here. As Igor explained, this would fail only in case bug in the code: either function signature is not correct or there is some bug in ethgo library and we should fail fast on such occasions as there is no point in handling error in this case.